src/intl/en/learn-quizzes.json

```
@@ -161,15 +161,15 @@
   "d001-c-explanation": "Web wallets have less security than hardware wallets because the private keys are stored on an internet-connected device.",
   "d001-d-label": "A desktop wallet",
   "d001-d-explanation": "Desktop wallets hold private keys on a computer hard drive, which typically has connections to the internet, and potentially compromised by other software.",
-  "d002-prompt": "From the options presented, which is the most secure way to store your seed phrase?",
+  "d002-prompt": "How should you store your seed phrase?",
   "d002-a-label": "In a photo on your phone",
   "d002-a-explanation": "This is not the most secure option. If this photo is uploaded to cloud storage then a hacker gets this image and gains access to your account.",
   "d002-b-label": "In a file on your computer",
   "d002-b-explanation": "This is not the most secure option. Hacker are increasingly looking for cryptocurrency related information on target devices. If a hacker accesses the file with your seed phrase they will gain access to your account.",
-  "d002-c-label": "Written down on paper",
-  "d002-c-explanation": "Of the available options, writing down your seed phrase on paper is the most secure.",
-  "d002-d-label": "In a text message to a trusted family member",
-  "d002-d-explanation": "You should never text your seed phrase to anyone. The message could be intercepted by a third party, and even if you trust this person absolutely, you do not know who may be able to access their phone.",
+  "d002-c-label": "In a text message to a trusted family member",
+  "d002-c-explanation": "You should never text your seed phrase to anyone. The message could be intercepted by a third party, and even if you trust this person absolutely, you do not know who may be able to access their phone.",
+  "d002-d-label": "None of the above",
+  "d002-d-explanation": "Your seed phrase should be secured in an offline manner. Things such as password managers or writing down on a piece of paper are suggested due to their offline nature.",
```

> Things such as password managers or writing down on a piece of paper are suggested due to their offline nature

Most (not all) password managers involve the internet/cloud syncing...

Maybe:

```suggestion
  "d002-d-explanation": "You seed phrase should be stored in a secure manner, ideally offline. Writing it down on paper is often recommended for this reason, but secure password managers are a good alternative.",
```